store/sqlstore/post_store.go

```diff
@@ -1976,7 +1976,7 @@ func (s *SqlPostStore) updateThreadsFromPosts(transaction *gorp.Transaction, pos
 		if thread, found := threadByRoot[rootId]; !found {
 			// calculate participants
 			var participants model.StringArray
-			if _, err := transaction.Select(&participants, "SELECT DISTINCT UserId FROM Posts WHERE RootId=:RootId", map[string]interface{}{"RootId": rootId}); err != nil {
+			if _, err := transaction.Select(&participants, "SELECT DISTINCT UserId FROM Posts WHERE RootId=:RootId OR Id=:RootId", map[string]interface{}{"RootId": rootId}); err != nil {
```

Comment:
Fixing something else here too, right? I think maybe this makes sense to be backported to 5.29

Reply:
@jespino  yup. same issue. the root posts author wasn't included in the participants

Reply:
once we are done with review, I'll create a backport PR for just these changes

Reply:
Great, that sounds good to me :)